portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = randomFact;
 }
+
+
+function getFromServlet(){
+    fetch('/data').then(response => response.text()).then((saying) => {
+    console.assert(document.getElementById('saying-container'), "You have no element with ID 'saying-container'"); 
```

Comment:
It's better to store the element in a variable.  That way if, for example, you change the name of your container you won't have to remember to change it in both places.